src/database/declarations.h

```diff
@@ -40,6 +40,21 @@ struct DataPosition : QPair<TableName, RowId> {
         : QPair<TableName, RowId>::QPair(table_name, row_id),
           tableName(first), rowId(second)
     {}
+    DataPosition(const DataPosition& other) = default;
+    DataPosition& operator=(const DataPosition& other) = default;
+};*/
+
+//[F]: How about something like this?
+struct DataPosition {
+    TableName tableName;
+    RowId rowId;
+    DataPosition()
+        : tableName(TableName())
+    {};
+    DataPosition(TableName table_name, RowId row_id)
+        : tableName(table_name), rowId(row_id)
+    {};
+
```

Comment:
The main problem is the debug integration with Qt. otherwise its what we want atm.

Reply:
fixed that